src/ios/KommunicateCordovaPlugin.swift

```diff
@@ -388,16 +259,46 @@ import Applozic
                     withPrechat = jsonObj["withPreChat"] as! Bool
                 }
                 
-                if jsonObj["isUnique"] != nil {
-                    self.isUnique = jsonObj["isUnique"] as! Bool
+                if jsonObj["isSingleConversation"] != nil {
+                    self.isUnique = jsonObj["isSingleConversation"] as! Bool
+                } else {
+                    self.isUnique = true
```

Comment:
all these else can be removed and move it to the top with the default values same place where 
self.createOnly = false is present

Reply:
Done